pkg/apis/config/feature_flags_test.go

```diff
@@ -53,6 +53,18 @@ func TestNewFeatureFlagsFromConfigMap(t *testing.T) {
 			},
 			fileName: "feature-flags-all-flags-set",
 		},
+		{
+			expectedConfig: &config.FeatureFlags{
+				EnableAPIFields: "alpha",
+				// These are prescribed as true by enabling "alpha" API fields, even
+				// if the submitted text value is "false".
+				EnableTektonOCIBundles: true,
+				EnableCustomTasks:      true,
```

Comment:
Nice!!

Should we also test that these two flags are `true` when they are configured to `true` and the api fields flag is set to `stable`?